ghost/i18n/locales/hi/portal.json

```diff
@@ -0,0 +1,159 @@
+    "Manage": "प्रबंधित करें",
+    "Maybe later": "शायद बाद में",
+    "Memberships unavailable, contact the owner for access.": "सदस्यता उपलब्ध नहीं है, पहुँच के लिए मालिक से संपर्क करें।",
+    "Monthly": "मासिक",
+    "More like this": "इस तरह के और",
+    "Name": "नाम",
+    "Need more help? Contact support": "और अधिक मदद चाहिए? समर्थन से संपर्क करें",
+    "Newsletters can be disabled on your account for two reasons: A previous email was marked as spam, or attempting to send an email resulted in a permanent failure (bounce).": "आपके खाते पर न्यूज़लेटर को दो कारणों से निष्क्रिय किया जा सकता है: एक पिछला ईमेल स्पैम के रूप में चिह्नित किया गया था, या एक ईमेल भेजने का प्रयास एक स्थायी विफलता (बाउंस) के कारण हुआ।",
+    "Not receiving emails?": "ईमेल प्राप्त नहीं हो रहे?",
+    "Now check your email!": "अब अपना ईमेल जांचें!",
+    "Once resubscribed, if you still don't see emails in your inbox, check your spam folder. Some inbox providers keep a record of previous spam complaints and will continue to flag emails. If this happens, mark the latest newsletter as 'Not spam' to move it back to your primary inbox.": "पुनः सदस्यता लेने के बाद, यदि आप अभी भी अपने इनबॉक्स में ईमेल नहीं देखते हैं, तो अपना स्पैम फ़ोल्डर जांचें। कुछ इनबॉक्स प्रदाता पिछली स्पैम शिकायतों का रिकॉर्ड रखते हैं और ईमेल को फ्लैग करना जारी रखेंगे। यदि ऐसा होता है, तो नवीनतम न्यूज़लेटर को 'स्पैम नहीं' के रूप में चिह्नित करें ताकि इसे वापस आपके मुख्य इनबॉक्स में ले जाया जा सके।",
+    "Permanent failure (bounce)": "स्थायी विफलता (बाउंस)",
+    "Plan": "योजना",
+    "Plan checkout was cancelled.": "योजना चेकआउट रद्द कर दिया गया।",
+    "Plan upgrade was cancelled.": "योजना उन्नयन रद्द कर दिया गया।",
+    "Please contact {{supportAddress}} to adjust your complimentary subscription.": "अपनी नि:शुल्क सदस्यता को समायोजित करने के लिए कृपया {{supportAddress}} से संपर्क करें।",
+    "Please fill in required fields": "कृपया आवश्यक फ़ील्ड भरें",
+    "Price": "कीमत",
+    "Re-enable emails": "ईमेल पुनः सक्षम करें",
+    "Recommendations": "सिफारिशें",
+    "Renews at {{price}}.": "{{price}} पर नवीनीकृत होता है।",
+    "Retry": "पुनः प्रयास करें",
+    "Save": "सहेजें",
+    "Send an email and say hi!": "ईमेल भेजें और नमस्ते कहें!",
+    "Send an email to {{senderEmail}} and say hello. This can also help signal to your mail provider that emails to and from this address should be trusted.": "{{senderEmail}} को एक ईमेल भेजें और नमस्ते कहें। यह आपके मेल प्रदाता को संकेत देने में भी मदद कर सकता है कि इस पते से और इस पते पर भेजे गए ईमेल पर विश्वास किया जाना चाहिए।",
+    "Sending login link...": "लॉगिन लिंक भेज रहा है...",
+    "Sending...": "भेज रहा है...",
```

Comment:
Sending.. : भेजा जा रहा है...
In the context of sending a mail, this will be more appropriate.  Like [this](https://github.com/TryGhost/Ghost/pull/20631/files#diff-ab4cf670bb4558a023d832ba8ef2e015a0f9b3101d82b6b9511c34711aa83580R56)

Reply:
Updated